chapters/teams.Rmd

```diff
@@ -473,10 +477,11 @@ SOFTWARE.
 
 The MIT license, the GPL, and the Hippocratic License are intended for use with software.
 When it comes to data and reports,
-the most widely used family of licenses are those produced by [Creative Commons][creative-commons].
+the most widely used family of licenses are those produced
+by [Creative Commons][creative-commons].\index{Creative Commons}
 These have been written and checked by lawyers and are well understood by the community.
 
-The most liberal option is referred to as [CC-0][cc_license],
+The most liberal option is referred to as [CC-0][cc_license],\index{Creative Commons!CC-0 license}\index{public domain}
```

Comment:
My gut says to include `\index{open source license!CC-0 license}` here, and similar for CC-BY below, to make them more discoverable.